frontend/src/pages/vpn-relay-welcome.page.tsx

```diff
@@ -0,0 +1,102 @@
+import { NextPage } from "next";
+import { useLocalization } from "@fluent/react";
+import { Layout } from "../components/layout/Layout";
+import { useRuntimeData } from "../hooks/api/runtimeData";
+import logo from "../components/layout/images/relay-logo.svg";
+import logoType from "../components/layout/images/fx-private-relay-premium-logotype-dark.svg";
+import panelArt from "./images/vpn-relay-panel-art.svg";
+import vpnLogo from "./images/mozilla-vpn-logo.svg";
+import styles from "./vpn-relay-welcome.module.scss";
+import { LinkButton } from "../components/Button";
+import { useProfiles } from "../hooks/api/profile";
+import { useRouter } from "next/router";
+import { isFlagActive } from "../functions/waffle";
+import { useEffect } from "react";
+import { authenticatedFetch } from "../hooks/api/api";
+
+const VpnRelayWelcome: NextPage = () => {
+  const { l10n } = useLocalization();
+  const profile = useProfiles();
+  const router = useRouter();
+  const runtimeData = useRuntimeData();
+
+  useEffect(() => {
+    // Tell the backend that there will probably be new subscriptions.
+    // It will then ask Firefox Accounts for up-to-date subscription information,
+    // and have that ready for the next request to `/profiles`.
+    authenticatedFetch("/accounts/profile/refresh");
+  }, []);
+
+  // redirect user if they haven't purchased vpn-relay
+  if (profile && profile.data && !profile.data[0].has_vpn) {
+    router.push("/");
+  }
+
+  return runtimeData && isFlagActive(runtimeData.data, "bundle") ? (
+    <Layout>
```

Comment:
```suggestion
    <Layout theme="premium">
```

Just like `/premium`, this page is related to a Premium feature - without this, the theme will be determined by the user's Premium status, which can have contract issues:

![image](https://user-images.githubusercontent.com/4251/190405326-1e30841e-a5e6-4258-993d-81fc483f1be6.png)